.github/workflows/push_images.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'feature/official_steamcmd'
+
+jobs:
+  docker:
+    strategy:
+      matrix:
+        include:
+           - imagename: Dockerfile.vanilla
+             tagname: vanilla
```

Comment:
Change this (or whatever case we decide) to `latest` to make it the default for `avivace/ror2server`